R/sjTabItemAnalysis.R

```diff
@@ -92,24 +92,22 @@
 #'             }
 #' 
 #' @examples
-#' # Data from the EUROFAMCARE sample dataset
-#' library(sjmisc)
-#' data(efc)
-#' 
-#' # retrieve variable and value labels
+#' Data from the EUROFAMCARE sample dataset
+#' library(sjmisc)data(efc)
+#' retrieve variable and value labels
 #' varlabs <- get_label(efc)
 #' 
 #' # recveive first item of COPE-index scale
 #' start <- which(colnames(efc) == "c82cop1")
 #' # recveive last item of COPE-index scale
 #' end <- which(colnames(efc) == "c90cop9")
-#'  
+#' 
 #' # create data frame with COPE-index scale
 #' mydf <- data.frame(efc[, c(start:end)])
 #' colnames(mydf) <- varlabs[c(start:end)]
-#' 
-#' \dontrun{
-#' sjt.itemanalysis(mydf)
+#' #' #'
+#' #' #' \dontrun{
```

Comment:
Sorry this is a mistake. My workflow was basically uncomment examples, use them for testing, comment out again. I guess in this case I forgot that. Is there a better way for testing?